server/swap/swap.go

```diff
@@ -17,21 +17,15 @@ import (
 	"decred.org/dcrdex/server/account"
 	"decred.org/dcrdex/server/asset"
 	"decred.org/dcrdex/server/coinlock"
+	"decred.org/dcrdex/server/coinwaiter"
 	"decred.org/dcrdex/server/comms"
 	"decred.org/dcrdex/server/matcher"
 )
 
-const (
-	// These constants are used for a chainWaiter to inform the caller whether
-	// the waiter should be run again.
-	tryAgain     = false
-	dontTryAgain = true
-)
-
 var (
-	// The chainWaiters will query for transaction data every recheckInterval.
+	// The coin waiter will query for transaction data every recheckInterval.
 	recheckInterval = time.Second * 5
-	// txWaitExpiration is the longest the Swapper will wait for a chainWaiter.
+	// txWaitExpiration is the longest the Swapper will wait for a coin waiter.
 	// This could be thought of as the maximum allowable backend latency.
 	txWaitExpiration = time.Minute
```

Comment:
Seems very similar to broacast timeout.  Would the same value be appropriate here, or maybe a multiple of it?

Reply:
The similarities are superficial, really. Network latency vs. community conduct. Linking the value to the broadcast timeout doesn't reflect its purpose very well, IMO. Could be an operator-set variable though.

Reply:
Let's make an issue then to track this.  I think this PR is g2g.